### PR TITLE
REL-870855 - [RIP] Remove 'EnableSyncNonDocumentFlowToggle' Toggle

### PR DESCRIPTION
**Pull request by sameerrelone**
- REL-870855 - Removed EnableSyncNonDocumentFlowToggle changes from all files i used backport ticket https://jira.kcura.com/browse/REL-808758
- Dev Verification Evidence 
[REL-870855 - RIP Smoke Verification.docx](https://github.com/user-attachments/files/16009885/REL-870855.-.RIP.Smoke.Verification.docx)

### CODE REVIEW

File: Source/kCura.IntegrationPoints.Web/Views/IntegrationPoints/Edit.cshtml [hunk 1]
```diff
@@ -68,11 +68,6 @@
             });
             return enableSyncNonDocumentFlowToggleValue;
         }
-
-        var promise = getEnableSyncNonDocumentFlowToggleValue();
-        promise.then(function (result) {
-            IP.data.params['EnableSyncNonDocumentFlowToggleValue'] = result;
-        });
```
Comment (shivarajkamadod) [inline Source/kCura.IntegrationPoints.Web/Views/IntegrationPoints/Edit.cshtml:75]: Please remove this method if it is not used:
var getEnableSyncNonDocumentFlowToggleValue = function() {
            var enableSyncNonDocumentFlowToggleValue = IP.data.ajax({
                url: IP.utils.generateWebAPIURL('ToggleAPI', 'kCura.IntegrationPoints.Common.Toggles.EnableSyncNonDocumentFlowToggle'),
                type: 'get'
            }).fail(function (error) {
                console.log(error);
            });
            return enableSyncNonDocumentFlowToggleValue;
        }

File: Source/kCura.IntegrationPoints.Web/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1091 @@
+{
```
Comment (shivarajkamadod) [inline Source/kCura.IntegrationPoints.Web/package-lock.json:1]: Remove this entire file, This file will generate runtime! Source/kCura.IntegrationPoints.Web/package-lock.json

**Review by shivarajkamadod — COMMENTED** [review]
Complete changes are looking good! Only few changes required to merge this PR

File: Source/kCura.IntegrationPoints.Web/Views/IntegrationPoints/Edit.cshtml (hunk 1 above)
Comment (sameerrelone) [reply Source/kCura.IntegrationPoints.Web/Views/IntegrationPoints/Edit.cshtml:75]: removed un-used method

File: Source/kCura.IntegrationPoints.Web/package-lock.json (hunk 2 above)
Comment (sameerrelone) [reply Source/kCura.IntegrationPoints.Web/package-lock.json]: removed

**Review by shivarajkamadod — APPROVED** [review]

**Review by shivarajkamadod — APPROVED** [review]

**Review by ravikonnur — APPROVED** [review]

**Pull request merged**